TestProject/TestProject/Entities/Models/List+CoreDataClass.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  List+CoreDataClass.swift
+//  TestProject
+//
+//  Created by Muhammad Taifoor Saleem on 28/01/2022.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(List)
+public class List: NSManagedObject {
+    
+    func myprint()
+    {
+        print("List Key: \(self.listKey)")
```

Comment:
Let's create a `InfoLogger` class and control logs from there.

Reply:
Done